manager-rs/configs/dauth-service/sample1.yaml

```diff
@@ -14,19 +14,33 @@ ed25519_keyfile_path: "./out/sample-network-1/dauth_service/ed25519_keys"
 database_path: "./out/sample-network-1/dauth_service/sqlite1.db"
 
 # Delay in seconds before tasks thread begins executing
-task_startup_delay: 5.0
+task_startup_delay: 1.0
 
 # Interval in seconds between checking for new tasks
-task_interval: 10.0
+task_interval: 1.0
+
+# The number of vector slices possible (also determines max backup networks)
+# Slice 0 is always reserved for the home network
+num_sqn_slices: 32
+
+# Max number of auth vectors a backup network should have at one time
+max_backup_vectors: 10
 
 # The set of users for this network (for testing)
 users: {
   "imsi-901700000000001": {
     "k": "465B5CE8B199B49FAA5F0A2EE238A6BC",
     "opc": "E8ED289DEBA952E4283B54E88E6183CA",
-    "sqn_max": "32",
-    "backup_network_ids": [
-      "sample-network-2"
-    ]
+
+    # Mapping of sqn slice to max sqn for that slice
+    "sqn_slice_max": {
```

Comment:
Do you mean the config as a whole or sqn_slice_max? The latter is mapping between sqn slice (i.e. 0-31 for the sims we have now) to the max value that a particular slice has seen (or the next available number on the slice). I needed a mapping like this so that I could initialize all in-use slices, since it seemed too limited to use one number to initialize all of them. I'm assuming 0 is used for the home network, and 1 is used back the only backup network.

Is this the wrong idea?